lib/challenge.js

```diff
@@ -33,6 +44,13 @@
 var pet_owner;
 
 // Answer Starts Here
+var pet_owner = {
+  name: Lauren Grant,
+  address: Somerville,
+  pet vaccinations: [
```

Comment:
Recall how object property names are strings. In object literal syntax, you're allowed to just write the property names without quotation marks, like so:

```js
var obj = {
    hasCookies : true
};
```

... but only so long as the property name consists entirely of valid name characters. This basically means "so long as it consists of nothing but consecutive letters". So, suppose we wanted to create an object literal with a  property named "favorite color". This would **not** work:

```js
var obj = {
    favorite color : "blue"
};
```

The interpreter would see that space between "favorite" and "color" and start looking for a colon (`:`). It won't find it, and it will conclude that you're using incorrect syntax. Here's how we reclaim these potential property names:

```js
var obj = {
    "favorite color" : "blue",
    "maybe-not" : false,
    "cookies+milk" : "yep",
    "!in&huh?" : null
};
```

In these cases, we have to wrap the property names in quotation marks. Once we do that, we're free to use whatever funny symbols we want, including spaces, hyphens, plus signs, and things we probably wouldn't even want to put in property names.